Source/ARTNSMutableDictionary+ARTDictionaryUtil.m

```diff
@@ -0,0 +1,17 @@
+//
+//  NSMutableDictionary+ARTNSMutableDictionary_ARTDictionaryUtil.m
+//  Ably
+//
+//  Created by Łukasz Szyszkowski on 29/06/2021.
+//  Copyright © 2021 Ably. All rights reserved.
+//
+
+#import "ARTNSMutableDictionary+ARTDictionaryUtil.h"
+
+@implementation NSMutableDictionary (ARTDictionaryUtil)
+
+- (void)addValueAsURLQueryItem:(NSString *)value forKey:(NSString *)key {
+    self[key] = [NSURLQueryItem queryItemWithName:key value:value];
+}
+
```

Comment:
I'm not sure wether setting dictionary value worth dedicated function, it's just does what dictionary can do by design. Only not to type `key` twice? I dunno.. Not critical though.

Reply:
it's similar but as you can see the value I wrapped in `NSURLQueryItem` so this "the sugar" for:
```objective-c
[dict setValue:[NSURLQueryItem queryItemWithName:"someKey" value:@"someValue"] forKey:@"someKey"];
```